### PR TITLE
add classmodifier to modal content

### PR DESCRIPTION
**Pull request by rox163**
### This project uses [semver](semver.org), please check the scope of this pr:
 - [x] #patch# - backwards-compatible bug fix
 - [ ] #minor# - adding functionality in a backwards-compatible manner
 - [ ] #major# - incompatible API change

# CHANGELOG
**adding** a classModifier for the contained modal.

### CODE REVIEW

**Comment by ember-cli-visual-acceptance** [comment]
# Visual Acceptance tests:
### No changes

**Comment by sglanzer-fire** [comment]
Sorry, I know I said this would be quick, but would you mind adding a test just to demonstrate that the modifier hits the right part of the DOM?

File: addon/components/dialogs/form.js [hunk 1]
```diff
@@ -43,7 +43,6 @@ export default FrostModalBinding.extend(PropTypesMixin, {
   getDefaultProps () {
     return {
       animation: form,
-      classModifier: 'form',
```
Comment (sglanzer-fire) [inline addon/components/dialogs/form.js:46]: I don't mind leaving this in (in case anyone is using it already)

**Review by sglanzer-fire — CHANGES_REQUESTED** [review]
I'd put the classModifier for form back, otherwise this is great, thanks!

**Comment by sglanzer-fire** [comment]
Approved

[Approved with PullApprove](https://pullapprove.com/ciena-frost/ember-frost-modal/pull-request/70/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ciena-frost/ember-frost-modal)

**Pull request merged**